plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/transition/DefaultViewportTransitionImpl.kt

```diff
@@ -49,9 +54,17 @@ internal class DefaultViewportTransitionImpl(
     completionListener: CompletionListener
   ): Cancelable {
     var isCancelableCalled = false
+    var animatorSet: AnimatorSet? = null
+    val startCamera = cameraDelegate.cameraState
+    val completedChildAnimators = mutableSetOf<CameraAnimatorType>()
+    var keepObserving = true
```

Comment:
pretty hard at least for me to understand the logic without going deep in general how viewport works but looks a bit strange that we declare local variable here. is it actually flag to understand that initial animation is still running? if yes, perhaps rename it

Reply:
no, this is a boolean we return inside the observer block to indicate if we want to receiver more updates from the data source. It was agreed with iOS team to use this pattern and ease the handling logic.